plugins/org.obeonetwork.m2doc/src/org/obeonetwork/m2doc/generator/TemplateProcessor.java

```diff
@@ -797,13 +804,13 @@ public AbstractConstruct caseLink(Link link) {
      * 
      * @param errorMessage
      *            the error message to set.
-     * @param run
+     * @param imageRun
```

Comment:
Why rename this parameter? The new name is wrong because this method is not only called when dealing with images, I think it's also used by the generation of tables for example.